services/ingestion/event_subscriber.go

```diff
@@ -224,7 +224,7 @@ func (r *RPCEventSubscriber) subscribe(ctx context.Context, height uint64) <-cha
 					// we can get not found when reconnecting after a disconnect/restart before the
 					// next block is finalized. just wait briefly and try again
 					time.Sleep(200 * time.Millisecond)
-				case codes.DeadlineExceeded, codes.Internal:
+				case codes.DeadlineExceeded, codes.Internal, codes.Unavailable:
```

Comment:
During some local testing with having 2 Flow Emulator processes, when I killed the 1st process, which was configured as the main AN(`AccessNodeHost`), then the EVM GW crashed with:

```bash
failure in event subscription with: recoverable: disconnected:
error receiving event: rpc error:
code = Unavailable desc = error reading from server: EOF
```

Adding the `codes.Unavailable` case, solved this issue.